docs/conf.py

```diff
@@ -34,7 +35,14 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.napoleon']
+extensions = [
```

Comment:
A couple of useful builtin sphinx extensions.
Most of them are likely to-be used in the (near) future, so might as well add them already.